index.qmd

```diff
@@ -150,14 +141,18 @@ for (thistrait in trait_groupings) {
 The APD Trait concepts express the traits defined by the APT.
 
 ```{r, results='asis', echo=FALSE, message=FALSE, warning=FALSE}
-list_of_traits <- traits_csv$identifier %>% paste0(base_url_traits, .)
+list_of_traits <- 
+  triples_with_labels %>% 
+    filter(Predicate == "http://www.w3.org/2000/01/rdf-schema#label") %>%
+    filter(stringr::str_detect(Subject, "trait_[:digit:]")) %>%
+    select(identifier = Subject)
 
 to_match_tmp <- triples_with_labels %>%
-  filter(Subject_stripped %in% list_of_traits) %>%
+  filter(Subject_stripped %in% list_of_traits$identifier) %>%
   filter(property == "label") %>%
   select(Subject_stripped, value)
  
-for (thistrait in list_of_traits) {
+for (thistrait in list_of_traits[[1]][1:nrow(list_of_traits)]) {
```

Comment:
Too complex!!!! 

`list_of_traits[[1]][1:nrow(list_of_traits)])` should be the same as `list_of_traits[[1]]`, the `1:n` is asking for all values, which seems unnecessary

Reply:
fixed